packages/components/src/styles/headline/headline.declaration.ts

```diff
@@ -0,0 +1,24 @@
+import type { Style } from '../../declaration';
+
+export default {
+  styleName: 'sd-headline',
+  summary:
+    'Headlines are vital for displaying content hierarchy and to improve accessibility. A headline can be additionally accompanied by an icon. The icon can be displayed on the left side or inline.',
+  status: 'experimental',
+  since: '1.7',
+  attributes: [
+    {
+      name: 'sd-headline--size-...',
+      options: ['3xl', 'xl', 'lg', 'base'],
+      description: "The headlines's size."
+    },
+    {
+      name: 'sd-headline--inverted',
+      description: 'Inverts the headline text.'
+    },
+    {
+      name: 'sd-headline--inline',
+      description: 'Sets inline behavior. Used exclusively when an sd-icon or other component is present.'
```

Comment:
what other components should be used with the headline using `inline`? If we dont have one, I'd suggest to remove the last part of the sentence to prevent people having this idea...

Reply:
sd-mark for example (see headline Samples). Setting inline here maintains the text wrapping for smaller screens. I chose to refer to components in general since inline can be pretty useful when a developer wants to combine headline with anything else.

Reply:
ok. Just asking as this might conflict with the expected usage from design. If they are fine with it, you can resolve my comment.